components/expenses/Expense.js

```diff
@@ -249,7 +249,9 @@ class Expense extends React.Component {
       (expense.status === 'PENDING' ||
         (expense.status === 'REJECTED' && Date.now() - new Date(expense.updatedAt).getTime() < 60 * 1000 * 15)); // we can approve an expense for up to 10mn after rejecting it
 
-    const canDelete = LoggedInUser && LoggedInUser.canPayExpense(expense) && expense.status === 'REJECTED';
+    const isCollectiveAdmin = LoggedInUser && LoggedInUser.canEditCollective(expense.collective);
```

Comment:
We use `props.collective` in the other checks, let's make it consistent

```suggestion
    const isCollectiveAdmin = LoggedInUser && LoggedInUser.canEditCollective(collective);
```